<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceRecognition</class>
 <widget class="QMainWindow" name="FaceRecognition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1476</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>人脸识别系统</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/images/icon.png</normaloff>:/images/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="Top" name="topWg" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>61</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>61</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="Left" name="leftWg" native="true">
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="middleWg" native="true">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>4</number>
           </property>
           <property name="topMargin">
            <number>4</number>
           </property>
           <property name="rightMargin">
            <number>2</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2" rowstretch="1,0,0">
             <item row="0" column="0">
              <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,0">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_18">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>4</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_17">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>27</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
background-image: url(:/images/bglb.png);</string>
                   </property>
                   <property name="text">
                    <string>  视频源</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="mainVideoGv">
                   <property name="sizePolicy">
                    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                     <horstretch>0</horstretch>
                     <verstretch>0</verstretch>
                    </sizepolicy>
                   </property>
                   <property name="minimumSize">
                    <size>
                     <width>300</width>
                     <height>200</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(0, 0, 0);</string>
                   </property>
                   <property name="frameShape">
                    <enum>QFrame::Box</enum>
                   </property>
                   <property name="verticalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                   </property>
                   <property name="horizontalScrollBarPolicy">
                    <enum>Qt::ScrollBarAlwaysOff</enum>
                   </property>
                   <property name="resizeAnchor">
                    <enum>QGraphicsView::AnchorViewCenter</enum>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_19">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <property name="bottomMargin">
                  <number>4</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_21">
                   <property name="minimumSize">
                    <size>
                     <width>0</width>
                     <height>30</height>
                    </size>
                   </property>
                   <property name="maximumSize">
                    <size>
                     <width>16777215</width>
                     <height>27</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">font: 12pt &quot;微软雅黑&quot;;
background-image: url(:/images/bglb.png);</string>
                   </property>
                   <property name="text">
                    <string>  日志列表</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QTextBrowser" name="logTB">
                   <property name="minimumSize">
                    <size>
                     <width>400</width>
                     <height>0</height>
                    </size>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">color: rgb(255, 0, 0);
font: 14pt &quot;微软雅黑&quot;;</string>
                   </property>
                   <property name="html">
                    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'微软雅黑'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:11pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item row="2" column="0">
              <widget class="FaceRecogResultList" name="resultWg" native="true">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>300</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_19">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>30</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>27</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">font: 12pt &quot;微软雅黑&quot;;
background-image: url(:/images/bglb.png);</string>
               </property>
               <property name="text">
                <string>  识别结果</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Top</class>
   <extends>QWidget</extends>
   <header>top.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Left</class>
   <extends>QWidget</extends>
   <header>left.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FaceRecogResultList</class>
   <extends>QWidget</extends>
   <header>facerecogresultlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
